### ISSUE TITLE
parameters which gave best results on multi speaker

### ISSUE THREAD

**Issue by Mega4alik** [kuleshov/audio-super-res#27]
Hi, can you please share with some training / data preparation parameters which gave best results? Most of them are not given in the paper or unclear.
For example: dimension (input vector size), stride step,  batch size, number of layers, learning rate, number of epochs in training 

Thanks in advance

**Comment by Sawyerb** [comment]
For the mutispeaker model with a downsampling ratio of 4, we used the following parameters:
patch size (input vector size): 8192
stride and spatial extent: 8
dilation factor: 2
upsampling and downsampling blocks: 4 
learning rate: 3e-4
number of epochs: 50

These parameters are described on page 6 of the paper: https://arxiv.org/pdf/1909.06628.pdf

Are there any other parameters that you're unsure about?
